lib/utils.pm

```diff
@@ -1402,6 +1403,25 @@ sub show_tasks_in_blocked_state {
     }
 }
 
+=head2 show_oom_info
+
+ show_oom_info
+
+Show logs about an out of memory process kill.
+
+=cut
+sub show_oom_info {
+    if (script_run('dmesg | grep "Out of memory"') == 0) {
+        my $oom = script_output('dmesg | grep "Out of memory"');
+        if (!check_var('BACKEND', 'svirt')) {
```

Comment:
can you use `has_ttys` from lib/Utils/Backends.pm?

Reply:
done